src/cloud-api-adaptor/pkg/adaptor/cloud/cloud.go

```diff
@@ -270,6 +271,13 @@ func (s *cloudService) CreateVM(ctx context.Context, req *pb.CreateVMRequest) (r
 		},
 	}
 
+	if daemonConfig.AuthJson != "" {
```

Comment:
I think  we also want to drop the authjson entry from the daemon.json I think, if we want to propagate it directly.